pallets/keystore/src/weights.rs

```diff
@@ -0,0 +1,18 @@
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::weights::{constants::RocksDbWeight, Weight};
+
+/// Weight functions for pallet_fennel_identity.
+pub trait WeightInfo {
```

Comment:
what's the point of WeightInfo here?

Reply:
It's required to implement benchmark-based weights in the actual lib.rs file on extrinsics definitions.

Reply:
Eventually it'll hold definitions of weights for each function as you see in https://github.com/fennelLabs/Fennel-Protocol/blob/milestone1-submission/pallets/trust/src/weights.rs.